app/assets/javascripts/experiments/table.js

```diff
@@ -226,7 +227,18 @@ var ExperimnetTable = {
         }
       });
 
+      if ($(`.my-module-selector[data-my-module="${id}"]`).prop('checked')) {
+        $(`.my-module-selector[data-my-module="${id}"]`).click();
```

Comment:
if you want clear row selection - `this.clearRowTaskSelection();`

Reply:
Yeah, because the behavior of the checkboxes is different depending on whether we want to archive from the header or the task row itself.

Thanks for the feedback.